pkg/policies/loader.go

```diff
@@ -44,20 +43,60 @@ const (
 
 // Loader defines the interface for policy loaders from contract attachments
 type Loader interface {
-	Load(context.Context, *v1.PolicyAttachment) (*v1.Policy, *v12.ResourceDescriptor, error)
+	Load(context.Context, *v1.PolicyAttachment) (*v1.Policy, *PolicyDescriptor, error)
+}
+
+// PolicyDescriptor Represents a policy reference. Used as FQ references.
+type PolicyDescriptor struct {
```

Comment:
so in the end you moved away from intoto descriptor why?

Reply:
This is internal. The final representation in the attestation is still an in-toto descriptor, but I didn't want to get locked into any structure in the internal logic for policies. Actually I think we should apply the same to the rest of the policies logic.

Reply:
oh, I get it, and agree!